packages/Search/examples/bvh_driver/bvh_driver.cpp

```diff
@@ -225,27 +162,27 @@ int main( int argc, char *argv[] )
         switch ( clp.parse( argc, argv, NULL ) )
         {
         case Teuchos::CommandLineProcessor::PARSE_ERROR:
-            rv = 1;
+            success = false;
         case Teuchos::CommandLineProcessor::PARSE_HELP_PRINTED:
         case Teuchos::CommandLineProcessor::PARSE_UNRECOGNIZED_OPTION:
         case Teuchos::CommandLineProcessor::PARSE_SUCCESSFUL:
             break;
         }
 
-        if ( rv )
+        if ( !success )
         {
             // do nothing, just skip other if clauses
         }
         else if ( node == "" )
         {
             typedef KokkosClassic::DefaultNode::DefaultNodeType Node;
-            rv = main_<Node>( clp, argc, argv );
+            main_<Node>( clp, argc, argv );
```

Comment:
If you are not using the return value from `main_`, might as well change it to `void`.